elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -1592,7 +1592,7 @@ private static BooleanSupplier getFipsSupplier() {
                     return false;
                 }
 
-                return isFips != null && isFips instanceof Boolean ? ((Boolean) isFips).booleanValue() : false;
+                return isFips != null && isFips instanceof Boolean ? (Boolean) isFips : false;
```

Comment:
There is a double check, instanceof null returns false:
"return isFips instanceof Boolean ? (Boolean) isFips : false;" is the same.

Reply:
The double check is intact, I just remove the unboxing.

Reply:
This seems to be a misunderstanding. I meant to remove the "isFips != null &&" because it's not necessary.

Reply:
Thanks! done now.

Reply:
Please check https://ci.wildfly.org (Guest login possible). If you push --force your existing builds will not be canceled and the new ones will enter the queue. If a previous build can show you an error (source code formatter, compile error, unit test fail) it is better to combine both the learning from the build and the recommendations here.
Other builds (maybe more important) have the chance to run and you avoid your build to be canceled.